addons/account/models/account_tax.py

```diff
@@ -288,21 +288,27 @@ def name_get(self):
         return name_list
 
     @api.model
-    def _search(self, args, offset=0, limit=None, order=None, count=False, access_rights_uid=None):
+    def _search(self, domain, offset=0, limit=None, order=None, access_rights_uid=None):
         context = self._context or {}
 
         if context.get('move_type'):
             if context.get('move_type') in ('out_invoice', 'out_refund'):
-                args += [('type_tax_use', '=', 'sale')]
+                domain = domain + [('type_tax_use', '=', 'sale')]
```

Comment:
```suggestion
                domain = expression.AND([domain, [('type_tax_use', '=', 'sale')]])
```
🤔

Reply:
This code is a relic from the past, and is actually no longer useful. No more worries :laughing: